web/src/beta/features/Dashboard/index.tsx

```diff
@@ -0,0 +1,92 @@
+import { FC } from "react";
+
+import { Typography } from "@reearth/beta/lib/reearth-ui";
+import { useT } from "@reearth/services/i18n";
+import { styled, useTheme } from "@reearth/services/theme";
+
+import ContentsContainer from "./ContentsContainer";
+import useHooks from "./hooks";
+import LeftSidePanel from "./LeftSidePanel";
+import { TabItems } from "./type";
+
+export type DashboardProps = {
+  workspaceId?: string;
+};
+
+export const tabsItem: Omit<TabItems[], "active"> = [
+  { id: "project", text: "Project", icon: "grid" },
+  { id: "asset", text: "Assets", icon: "file" },
+  { id: "members", text: "Members", icon: "appearance" },
+  { id: "bin", text: "Recycle bin", icon: "trash" },
+  { id: "plugin", text: "Plugin Playground", icon: "puzzlePiece", path: " " },
+  { id: "documentary", text: "Documentary", icon: "book", path: " " },
+  { id: "community", text: "Community", icon: "usersFour", path: " " },
+  { id: "help", text: "Help & Support", icon: "question", path: " " },
+];
```

Comment:
I think it's better to define them seprately as topTabs and bottomTabs here, i saw you're doing somthing like .slice(3) .slice(4) with this list just to separete them into two parts, this way could easlisy lead to mistake when we change this list in the futrue.
And for the bottom, actually they are only links, maybe we should not regard them as tab.